plugin/evm/header/gas_limit_test.go

```diff
@@ -140,3 +144,41 @@ func VerifyGasLimitTest(t *testing.T, feeConfig commontype.FeeConfig) {
 		})
 	}
 }
+
+func TestGasCapacity(t *testing.T) {
+	t.Run("normal", func(t *testing.T) {
+		GasLimitTest(t, testFeeConfig)
+	})
+	t.Run("double", func(t *testing.T) {
+		GasLimitTest(t, testFeeConfigDouble)
+	})
+}
+
+func GasCapacityTest(t *testing.T, feeConfig commontype.FeeConfig) {
```

Comment:
this is unused 🤔

Reply:
ah good catch, should have been used by `TestGasCapacity`